BranchMetrics/android-branch-deep-linking-attribution
# chore(deps):SDK-1991 -- [Android] Update Play Store Test Dependency

## Reference
SDK-1991 -- [Android] Update Play Store Test Dependency

## Description
Our testbed app is currently using `com.google.android.gms:play-services-ads:16.0.0` which is both out of date and not suggested one for our use case of obtaining the aaid. It just so happens that they share the same classes which is why we’re able to invoke by reflection. This adds to confusion for developers.

Replacing with `com.google.android.gms:play-services-ads-identifier:18.0.1` which has been in our official documentation.

Additional details https://github.com/BranchMetrics/capacitor-branch-deep-links/issues/111#issuecomment-1609065949

## Testing Instructions
Checkout the branch- verify AAID is still present. Test and automation validated.

## Risk Assessment [`LOW`]



- [✅] I, the PR creator, have tested — integration, unit, or otherwise — this code.

## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style

cc @BranchMetrics/saas-sdk-devs for visibility.
